modules/snap_community_charts/js/charts.js

```diff
@@ -267,7 +304,9 @@ snapCharts = {
 				community: snapCharts.data.community,
 				dataset: snapCharts.data.dataset,
 				scenario: snapCharts.data.scenario,
-				variability: snapCharts.data.variability
+				variability: snapCharts.data.variability,
+        baseline: snapCharts.data.baseline
+        //resolution: snapCharts.data.resolution
```

Comment:
This was commented out because setting the resolution was unnecessary with the choice between PRISM and CRU 3.2 data being the determining factor between resolution being either 2km or 10min respectively. I need to clean up those loose ends in regards to commented out code.